README.md

```diff
@@ -19,36 +19,31 @@ See the ```Server.configureFrontend()``` method for details.
 The webpack server is implemented in [src/webpack.ts](src/webpack.ts).
 
 #### Templates
-You can provide your templates to the method which controls the HTML served for the frontend.
-
-To override the main HTML template, you can provide a function to the ```renderMain``` option.
-
-The function will be called with the request, response and the template parameters.
+The templates are managed via [express template engine](https://expressjs.com/en/guide/using-template-engines.html).
+In short, express needs to know where the views live, and which renderer to use.
 
 Make sure to include all parameters in the template, otherwise the block will not work as expected.
+
 ```typescript
 // The directory where the output of the build is stored
 const DIST_DIR = Path.resolve(__dirname, "../dist");
 const webpackConfig = require("../webpack.development.config");
-server.configureFrontend(DIST_DIR, webpackConfig, {
-    renderMain(req: Request, res: Response, params: MainTemplateParams): string {
-        return `<!DOCTYPE html>
-            <html lang="en-US">
-              <head>
-                <title>My site</title>
-                <meta charset="utf-8" />
-                <base href="${params.baseUrl}" />
-                <link rel='shortcut icon' type="image/svg+xml" href='/assets/images/favicon.svg' />
-                ${params.styles}
-              </head>
-              <body>
-                ${params.scripts}
-              </body>
-            </html>`;
-    }
+server.configureFrontend(DIST_DIR, webpackConfig);
+
+server.express().set('views', Path.resolve(__dirname, "../templates"));
```

Comment:
So I guess the point would be to generate this code so they wont need to figure this out?

Reply:
Yea it should be part of the template for sure.

This whole setup feels like it could be a separate package, since it's solving an isolated problem.